kolibri/core/assets/src/views/CoreBase/index.vue

```diff
@@ -87,7 +87,9 @@
         <AppError />
       </KPageContainer>
 
-      <slot v-else></slot>
+      <div v-else id="main" role="main" tabindex="-1" class="main">
+        <slot></slot>
+      </div>
```

Comment:
This seems fine, but curious whether we could just do

```html
<slot v-else id="main" role="main" tabindex="-1" class="main"></slot>
```

Reply:
Nope, doesn't work. I think the only API slots expose is `v-slot`.